docs/src/troubleshooting.md

```diff
@@ -0,0 +1,21 @@
+# Troubleshooting and Workflow Tips
+
+## High Failure Rate
+
+While some EKI variants include failure handlers, excessively high failure rates (i.e., > 80%) can lead to inversions finding local minima or failing to converge. To address this:
+
+- **Stabilize the Forward Model**: Ensure the forward model remains stable for small parameter perturbations in offline tests.
+- **Adjust Priors**: Reduce the uncertainty in priors. Priors with large variances can lead to forward evaluations that deviate significantly from the known prior means, increasing the likelihood of failures.
+- **Increase Ensemble Size**: Without localization (or other methods that break the subspace property), the ensemble size should generally exceed the number of parameters being optimized. The ensemble size needs to be large enough to ensure a sufficient number of successful runs, given the failure rate.
+- **Increase Preconditioner Retry Count**: If using a preconditioner that recursively draws from the prior until a stable run is achieved, increase the retry count (typically to more than 5).
```

Comment:
Is this something in EKP? If not maybe you are _suggesting_ use of a preconditioner that recursively draws from the prior. Note this is inefficient

Reply:
it's not natively in EKP. will modify the description to reflect that.